core/src/shred_fetch_stage.rs

```diff
@@ -24,7 +23,9 @@ use {
     },
 };
 
-const DEFAULT_LRU_SIZE: usize = 10_000;
+const DEDUPER_FALSE_POSITIVE_RATE: f64 = 0.001;
+const DEDUPER_NUM_BITS: u64 = 637_534_199; // 76MB
```

Comment:
what's the order of increase over the `LruCache` deduper?

Reply:
The effective capacity of the bloom-deduper is 10M.
So 1000x compared with the old lru-cache.

In terms of reset cycle, the old reset cycle was `DEFAULT_MS_PER_SLOT` but now 5min.

Reply:
sorry i meant the ram usage

Reply:
`637_534_199` bits is 76MB.
Previously 10k u64, which is 78KB, plus the overhead of an lru-cache.

Reply:
k, so still about 1000x. thanks!